extensions/EXT_disjoint_timer_query/extension.xml

```diff
@@ -224,25 +224,29 @@ interface EXT_disjoint_timer_query {
         ext.endQueryEXT(ext.TIME_ELAPSED_EXT);
 
         // ...at some point in the future, after returning control to the browser and being called again:
-        var available = ext.getQueryObjectEXT(query, ext.QUERY_RESULT_AVAILABLE_EXT);
-        var disjoint = gl.getParameter(ext.GPU_DISJOINT_EXT);
+        let available = ext.getQueryObjectEXT(query, ext.QUERY_RESULT_AVAILABLE_EXT);
+        let disjoint = gl.getParameter(ext.GPU_DISJOINT_EXT);
 
         if (available &amp;&amp; !disjoint) {
           // See how much time the rendering of the object took in nanoseconds.
-          var timeElapsed = ext.getQueryObjectEXT(query, ext.QUERY_RESULT_EXT);
+          let timeElapsed = ext.getQueryObjectEXT(query, ext.QUERY_RESULT_EXT);
 
           // Do something useful with the time.  Note that care should be
           // taken to use all significant bits of the result, not just the
           // least significant 32 bits.
           adjustObjectLODBasedOnDrawTime(timeElapsed);
+
+          // Clean up the query object.
+          ext.deleteQueryEXT(query);
```

Comment:
`createQueryEXT` was called regardless of this conditional, so shouldn't it be deleted unconditionally?

Same everywhere else.

Reply:
Looking at the example code (it's hard to do this without merging in these changes and building the extensions), it's at this point, inside this if-clause, that the query object's finally resolved, and the application's done with it. I think it would be confusing to put the deletion of the object anywhere else, since this code's basically run in a polling loop.

Reply:
Ah, hang on - I see what you mean. Yes, let me revise this.

Reply:
Code substantially reworked in the latest commit. Please re-review. Thanks.